src/components/TheHeader.vue

```diff
@@ -1,10 +1,20 @@
 <template>
   <header>
-    <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar data-cy="nav-drawer" :color="theme === 'light' ? 'surface-2' : 'surface-1'">
       <v-toolbar-title>
         <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
-          <TheUserMenu />
+          <TheUserMenu :theme="theme" />
           <img
+            v-if="theme === 'light'"
+            id="logo"
+            alt="Cuttle logo"
+            src="/img/cuttle_logo_text_brown.svg"
+            width="60"
+            height="60"
+            class="ma-md-auto"
+          >
+          <img
+            v-else
             id="logo"
             alt="Cuttle logo"
             src="/img/cuttle_logo_text_white.svg"
```

Comment:
How about we use a single `<img>` tag with and make the src dynamic? All the other fields are the same so it'd by more DRY that way